ep_gui/resources/lang/ep_gui.en-US.rc

```diff
@@ -362,16 +362,16 @@ BEGIN
     IDS_OTHER_PWRBTNACTION_2    "Shut down (default)"
     IDS_OTHER_PWRBTNACTION_4    "Restart"
     IDS_OTHER_NOREDIRECT        "Prevent the following Control Panel links from being redirected to the Settings app:"
-    IDS_OTHER_NOREDIRECT_SYSTEM "System / About page"
-    IDS_OTHER_NOREDIRECT_PROGRAMS "Programs and Features"
-    IDS_OTHER_NOREDIRECT_DATETIME "Adjust date/time"
-    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize notification icons"
+    IDS_OTHER_NOREDIRECT_SYSTEM "System/About page"
+    IDS_OTHER_NOREDIRECT_PROGRAMS "Programs && Features"
+    IDS_OTHER_NOREDIRECT_DATETIME "Adjust Date && Time"
+    IDS_OTHER_NOREDIRECT_TRAYICONS "Customize Notification-icons"
 
     IDS_UPDATES                 "Updates"
 
     IDS_UPDATES_POLICY          "When File Explorer starts"
-    IDS_UPDATES_POLICY_1        "Notify about available updates (default)"
-    IDS_UPDATES_POLICY_0        "Prompt to install available updates"
+    IDS_UPDATES_POLICY_1        "Only notify when updates are available (default)"
+    IDS_UPDATES_POLICY_0        "Notify in all cases, updates available or no updates available"

```

Comment:
"Notify when available updates should be installed" or similar makes more sense.

Reply:
Perhaps it is better to write "Warn in all cases, presence or absence of update"